<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>361</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Light</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search for Starting City</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_search_start">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="input_list_start">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_search_end">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>290</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="input_list_end">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>340</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="input_calculate">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>410</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>361</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Light</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search for Ending City</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>371</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>503</x>
          <y>0</y>
          <width>20</width>
          <height>531</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>371</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLCDNumber" name="output_distance">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>460</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>440</y>
          <width>191</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Light</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Shortest Distance:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>390</y>
          <width>61</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Light</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Units:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="input_units">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>671</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="output_progress">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>21</width>
          <height>481</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShortCut">
    <property name="title">
     <string>ShortCut</string>
    </property>
   </widget>
   <addaction name="menuShortCut"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
